weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/FeedbackGrid.tsx

```diff
@@ -62,7 +62,11 @@ export const FeedbackGrid = ({
     );
     // only keep the most recent feedback for each (feedback_type, creator)
     const combinedFiltered = Object.values(combined).map(
-      fs => fs.sort((a, b) => b.created_at - a.created_at)[0]
+      fs =>
```

Comment:
more correct type checking caught this error in the annotations path

Reply:
ah, nice